.github/renovate.json

```diff
@@ -12,7 +12,7 @@
     "github.com/rancher/hull",
     "github.com/rancher/lasso",
     "github.com/rancher/wrangler",
-    "github.com/rancher/wrangler/v2",
```

Comment:
Unless I'm wrong - because Renovate config here applies to the whole repo we want to keep v2 listed for the Rancher 2.8 branch of BRO. Also this change would create 2 instances of V3 wrangler.

Reply:
We probably want both `v2` and `v3` here because these list dependencies to ignore.